src/SLCore.UnitTests/Common/Helpers/ConnectionIdHelperTests.cs

```diff
@@ -20,12 +20,26 @@
 
 using System;
 using SonarLint.VisualStudio.SLCore.Common.Helpers;
+using SonarLint.VisualStudio.TestInfrastructure;
 
 namespace SonarLint.VisualStudio.SLCore.UnitTests.Common.Helpers;
 
 [TestClass]
 public class ConnectionIdHelperTests
 {
+    [TestMethod]
+    public void MefCtor_CheckIsExported()
+    {
+        MefTestHelpers.CheckTypeCanBeImported<ConnectionIdHelper, IConnectionIdHelper>();
+    }
```

Comment:
What's your opinion on arrow methods and their general adoption in the codebase?

Reply:
I am pro-arrow methods in general, but in the case of tests I am against it